### ISSUE TITLE
Tetrapendant compounds

### ISSUE THREAD

**Issue by MFernflower** [OpenSourceTB/OSTB_Series4_OpenCyclams#2]
These should be easy to make as the tetrapropargyl precursor could likely be made in a solvent-less fashion:
![CORE111](https://user-images.githubusercontent.com/3164942/121380087-97e69f00-c912-11eb-84eb-479e42e00403.png)

![tetrapendant](https://user-images.githubusercontent.com/3164942/121440632-46f89a00-c956-11eb-9fd1-2ea37e0a637f.png)

**Comment by MFernflower** [comment]
@mattodd @KlementineJBS

**Comment by MFernflower** [comment]
This then of course can be reacted with 1-Azidonaphthalene to form a novel but likely not very water soluble compound 

My first proposed compounds in #1 dont seem to be too interesting as per the paper-- hopefully this tetrapendant compound will prove worthy

**Comment by MFernflower** [comment]
@KlementineJBS Is something like this being considered for synthesis?

**Comment by KlementineJBS** [comment]
Yes, we are considering this kind of thing - though currently still working on trying to do the [boc-protected bis- versions](https://github.com/KlementineJBS/ELN_USYD_Honours/issues/22). 

Another researcher successfully made a tetrapendant compound from a 1,8-diaminooctane scaffold, but found it had a >100 micromolar MIC against the H37Rv _Mtb_ strain. Unfortunately most of my own lab work has ground to a halt due to severe covid-19 restrictions, so it may take some time until we can get onto making this kind of compound ourselves on different scaffolds...

Also, when you say it can be made in a solvent-less fashion - could you explain what you mean? I haven't seen that kind of scheme before.

**Comment by MFernflower** [comment]
@KlementineJBS I was assuming that at a small scale the tetra-propargyl precursor compound (to the feed into the click rxn) could be made in solvent less fashion as the reagents to make it are liquids (propargyl bromide and the diamine)

**Comment by KlementineJBS** [comment]
Oh! I get what you mean now and that makes a lot of sense - I've been struggling over what solvent to use so definitely will consider giving this a go when I can get back in the lab.
